main/network.c

```diff
@@ -430,30 +430,27 @@ php_socket_t php_network_bind_socket_to_local_addr(const char *host, unsigned po
 	for (sal = psal; *sal != NULL; sal++) {
 		sa = *sal;
 
-		/* create a socket for this address */
-		sock = socket(sa->sa_family, socktype, 0);
-
-		if (sock == SOCK_ERR) {
-			continue;
-		}
-
 		switch (sa->sa_family) {
 #if HAVE_GETADDRINFO && HAVE_IPV6
 			case AF_INET6:
-				((struct sockaddr_in6 *)sa)->sin6_family = sa->sa_family;
```

Comment:
That doesn't seem to make sense, it is equal to `a = a`, so I removed all of such code, correct me if I am wrong.